java/ql/lib/semmle/code/java/controlflow/internal/Preconditions.qll

```diff
@@ -11,6 +11,8 @@ import java
  * is equal to `checkTrue` and throws otherwise.
  */
 predicate conditionCheckMethod(Method m, boolean checkTrue) {
+  conditionCheckMethod(m, 0, checkTrue)
```

Comment:
I was thinking about doing that, but because this was a public API I didn't want to beak any downstream consumers. That being said, this is in a package called `internal` so I'm happy to make that change.

Reply:
Good point, keep the two-arg overload to be non-breaking (actually consider giving this a different name; that's usually better than overloading for clarity's sake), but just let that one be a convenience method with a one-line definition rather than a mutually recursive pair.

Reply:
> actually consider giving this a different name;

Got any suggestions here?

Reply:
`conditionCheckMethodArgument`?